app/main/helpers/suppliers.py

```diff
@@ -46,3 +46,23 @@ def supplier_company_details_are_complete(supplier_data):
              for f in
              contact_required_fields])
     )
+
+
+def get_company_details_from_supplier(supplier):
+    address = {"country": supplier.get("registrationCountry")}
```

Comment:
Are we actually using `country` in the `legal_text` template? I ask because in the DB this field is stored in the format `"country:GB"`, which needs transforming via a template filter before use (see [this example in the Admin](https://github.com/alphagov/digitalmarketplace-admin-frontend/blob/master/app/templates/suppliers/_company_details_table.html#L12)).

If we're not using it, great - let's delete 😸

Reply:
We are not using it currently. It's unclear from the prototype what format/resolution of address is needed. I can delete it or transform it as you say.

@NoraGDS do you know if country is required in the address displayed in the contract?

Reply:
Yes, it probably is. EU and US-based companies are often suppliers.

Reply:
I've pushed a commit to fix.

Reply:
We can leave this for now I think.